README.md

```diff
@@ -9,7 +9,7 @@ Feedstock license: [BSD-3-Clause](https://github.com/conda-forge/cpptango-feedst
 
 Summary: Tango-Controls C++ library
 
-Development: https://github.com/tango-controls/cppTango
+Development: https://gitlab.com/tango-controls/cppTango
 
 Documentation: https://tango-controls.github.io/cppTango-docs
```

Comment:
```suggestion
Documentation: https://tango-controls.gitlab.io/cppTango
```

Reply:
To update the README, we should rerender. I should have after changing the doc url. I just did.